site/en/setup.md

```diff
@@ -166,23 +189,43 @@ made in containers back to your local Docker image.)
 
 ### Testing the Docker installation
 
-To start, download and extract a document from the [OCR-D GT Repo](https://ocr-d-repo.scc.kit.edu/api/v1/metastore/bagit/):
+To start, download and extract a document from the [OCR-D GT Repo](https://ola-hd.ocr-d.de/search?q=&fulltextsearch=false&metadatasearch=false&isGT=true&perPageRecords=30):
 
 ```sh
-wget 'https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/736a2f9a-92c6-4fe3-a457-edfa3eab1fe3/data/wundt_grundriss_1896.ocrd.zip'
-unzip wundt_grundriss_1896.ocrd.zip
+wget "https://ola-hd.ocr-d.de/api/export?id=21.T11998/0000-001C-F82E-8&internalId=false" -O wundt_grundriss_1896.ocrd.zip
+sudo unzip wundt_grundriss_1896.ocrd.zip
```

Comment:
```suggestion
unzip wundt_grundriss_1896.ocrd.zip
```

Why the `sudo`?

Reply:
I ran into a PermissionError and was only able to unzip with `sudo`, so I thought it would not hurt to add sudo here just to be sure. but I can't reproduce that ATM. I'll come back to it if I run into that again. Probably not necessary to add it here.